File: app/src/main/java/pl/pola_app/internal/di/WidgetModule.java
package pl.pola_app.internal.di;

import android.content.Context;
import android.content.res.Resources;

import dagger.Module;
import dagger.Provides;

@Module
public class WidgetModule {
    private final Context context;

    public WidgetModule(Context context) {
        this.context = context;
    }

    @Provides
    Resources provideResources() {
        return context.getResources();
    }
}

File: app/src/main/java/pl/pola_app/internal/di/WidgetModule.kt
package pl.pola_app.internal.di

import android.content.Context
import android.content.res.Resources
import dagger.Module
import dagger.Provides

@Module
class WidgetModule(private val context: Context) {
    @Provides
    fun provideResources(): Resources {
        return context.resources
    }
}